test/mock-server/responses.ts

```diff
@@ -0,0 +1,305 @@
+export default {
+  telephony: {
+    v1: {
+      cross_device_sms: { status: 'OK' },
+    },
+  },
+  api: {
+    v3: {
+      documents: {
+        cut_off: {
+          id: 'f5c8a78f-aa96-4803-8e55-022ec9db946e',
+          created_at: '2020-11-26T15:47:52Z',
+          file_name: 'identity_card_with_cut-off.png',
+          file_type: 'jpg',
```

Comment:
Minor: Doesn't affect any of the tests AFAIK, but the file type here doesn't match the file name's file extension.